packages/parse5/lib/index.ts

```diff
@@ -7,6 +7,7 @@ export { type DefaultTreeAdapterMap, defaultTreeAdapter } from './tree-adapters/
 export type { TreeAdapter, TreeAdapterTypeMap } from './tree-adapters/interface.js';
 export { type ParserOptions, /** @internal */ Parser } from './parser/index.js';
 export { serialize, serializeOuter, type SerializerOptions } from './serializer/index.js';
+export { ERR as ErrorCodes } from './common/error-codes.js';
 export type { ParserError } from './common/error-codes.js';
```

Comment:
Can these lines be combined? Something like

```ts
export { ERR as ErrorCodes, type ParserError } from ...
```

Reply:
fixed